<launch>
  <arg name="m_init_yaw" default="3.0"/>

    <node pkg="localization" type="EKF_node" name="EKF_node"  output="screen">
  </node>
        <param name="init_yaw" type="double" value="$(arg m_init_yaw)" />
</launch>